source/host.c

```diff
@@ -72,7 +72,9 @@ cvar_t	teamplay = {"teamplay","0",false,true};
 cvar_t	samelevel = {"samelevel","0"};
 
 cvar_t	show_fps = {"show_fps","0", true};	// set for running times - muff
+#ifndef __WII__
```

Comment:
Can you justify this?

Reply:
Wii homebrew is required to have vsync enabled which makes this FPS cap redundant, and also creates a timing issue within the Dolphin emulator due to changes in Host_FilterTime

Reply:
Please document this then with your ifdef. Thanks!